### PR TITLE
Mod0

### PR DESCRIPTION
**Pull request by RicardoRojo**
Added models and created tests to complete the assignments for course 3 week 1 until topic 10

### CODE REVIEW

File: app/views/videos/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+%article.video_category
+  - @categories.each do |cat|
```
Comment (alagram) [inline app/views/videos/index.html.haml]: Tiny nitpick nut you could change the variable name to something like `category` instead if `cat`

File: app/views/videos/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+%article.video_category
+  - @categories.each do |cat|
+    %header
+      %h3= link_to cat.name, category_path(cat)
+    .videos.row
+      - cat.videos.each do |vid|
```
Comment (alagram) [inline app/views/videos/index.html.haml]: You could also do `video` here.

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: myflix_development
+  pool: 5
+  username: boottest
+  password: Iputo*34
```
Comment (alagram) [inline config/database.yml:7]: Not a big deal now, but you'll learn how to handle such sensitive info later in the course

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  username: boottest
+  password: Iputo*34
+
+test:
+  adapter: postgresql
+  encoding: unicode
+  database: myflix_test
+  pool: 5
+  username: boottest
+  password: Iputo*34
```
Comment (alagram) [inline config/database.yml:15]: Same here.

**Comment by alagram** [comment]
Overall, well done!

File: app/views/videos/index.html.haml (hunk 1 above)
Comment (RicardoRojo) [inline app/views/videos/index.html.haml]: Thanks, just a little bit of laziness.:). I´ll correct

File: app/views/videos/index.html.haml (hunk 2 above)
Comment (RicardoRojo) [inline app/views/videos/index.html.haml]: Same here.

File: config/database.yml (hunk 3 above)
Comment (RicardoRojo) [inline config/database.yml:7]: Thanks @alagram.
You are right. Didnt consider it was a problem, I suppose the solution is including it in the gitignore file. is that right?. 
By the way, what about deploying to heroku?. Do you have to include a production section in the file? Are the databases automatically created or you need to include database name, user and password?

**Comment by RicardoRojo** [comment]
Thanks very much for your comments @alagram .I´ll review my code with the suggested changes and will keep working on the project.

File: config/database.yml (hunk 3 above)
Comment (alagram) [inline config/database.yml:7]: Not a big deal but yes you'd normally want to use `.gitignore` for this. You'll also learn how to use environment variables later in the course to handle sensitive information. You don't need to worry about heroku deployment (production database) as it'll be handled automatically when you deploy.

**Pull request closed without merging**